src/docs/output-targets/docs-readme.md

```diff
@@ -27,12 +27,16 @@ export const config: Config = {
 };
 ```
 
-Another option would be to add the flag `--docs-readme`, such as:
+Another option would be to use the `--docs` CLI flag, like so:
 
 ```bash
-stencil build --docs-readme
+stencil build --docs
 ```
 
+This will cause the Stencil compiler to automatically add a `docs-readme`
+output target to your configuration, without having to save it in
+`stencil.config.ts`.
+
```

Comment:
This sentence may be a little confusing - it states it adds a docs-readme output target to your configuration, but could be a little clearer when it comes to this not being persisted in your `stencil.config.ts`.  What do you think about:

```suggestion
This will cause the Stencil compiler to perform a one-time generation of README
files using the [`docs-readme`](https://stenciljs.com/docs/docs-readme) output target.
```